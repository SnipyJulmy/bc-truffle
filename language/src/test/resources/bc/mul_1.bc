31231 * 72319873321;
9 * 0;
0 * 9;
0 * 0;
0 * -1;
0 * -2;
0 * -3;
-1 * 0;
-3 * 0;
-3 * -4;
halt
