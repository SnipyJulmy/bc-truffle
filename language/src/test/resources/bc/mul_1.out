2258621963688151
0
0
0
0
0
0
0
0
12
